emma-language/src/main/scala/org/emmalanguage/ast/AST.scala

```diff
@@ -183,4 +183,6 @@ trait AST extends CommonAST
     if defs.size > 1
     dfn <- defs.tail
   } yield dfn
+
+  private[ast] def freshNameSuffix: String
```

Comment:
:+1: for returning a `Char`. If a more complex suffix is needed later, changing the type won't affect use sites.